python/servo/mutation/test.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2013 The Servo Project Developers. See the COPYRIGHT
+# file at the top-level directory of this distribution.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import fileinput
+import re
+import subprocess
+import sys
+import os
+import random
+DEVNULL = open(os.devnull, 'wb')
+
+
+def mutate_random_line(file_name):
```

Comment:
This function will scale better to adding more mutation strategies later if you avoid hardcoding this one strategy right now. In future work, I'd like to see it take the regex to match (here `r'\s&&\s'`) and the string to replace it with (here `' || '`) as arguments, so that later we can have a whole list of arbitrary strategies and just pass them in one at a time (or even generate them from a list of regexes and a list of strings on the fly if we're feeling fancy). 

For now, just hardcoding the regex and string where you call the function, instead of within the function's implementation, would be perfectly fine.

Reply:
Thank you for your comment. Our Mutation test project was split up into 2 separate checkpoints. Checkpoint 1 is done and we are currently working on checkpoint 2.

The next set of changes addresses this comment.
Here is one such change -
https://github.com/dsandeephegde/servo/commit/e09464e7620158a1d07cd038c6bac3368f442d69

Some of the other changes planned are included from the subsequent steps in - https://github.com/servo/servo/wiki/Mutation-testing-project